ansible/roles/osx-kvm/templates/macOS-libvirt-Catalina.xml.template

```diff
@@ -0,0 +1,186 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Reuse an existing template.

'Software Maintainability' is a thing.

Reply:
You can always just remove the original xml, since this one supersedes it.

Reply:
Why not merge back the changes from this newly introduced file into the old one and test the whole thing rigorously?

'Reliability' and 'Reproducibility' (from scratch) are the biggest feature of this repository. Even after the attention to these aspects, I know that things are not perfect (our `config.plist` is terrible) but hey let's keep progressing in the right direction :)